.goreleaser.yml

```diff
@@ -3,84 +3,98 @@ before:
-  use: buildx
-  build_flag_templates:
-    - "--platform=linux/ppc64le"
-    - "--cache-from"
-    - "type=gha,scope=ppc64le,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
-    - "--builder={{ .Env.BUILDX_BUILDER }}"
-  extra_files:
-    - "images/ansible-operator/Pipfile"
-    - "images/ansible-operator/Pipfile.lock"
-- image_templates:
-    - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-s390x"
-  dockerfile: images/ansible-operator/Dockerfile
-  goos: linux
-  goarch: s390x
-  use: buildx
-  build_flag_templates:
-    - "--platform=linux/s390x"
-    - "--cache-from"
-    - "type=gha,scope=s390x,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
-    - "--builder={{ .Env.BUILDX_BUILDER }}"
-  extra_files:
-    - "images/ansible-operator/Pipfile"
-    - "images/ansible-operator/Pipfile.lock"
+  - image_templates:
+      - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-amd64"
+    dockerfile: images/ansible-operator/Dockerfile
+    goos: linux
+    goarch: amd64
+    use: buildx
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--cache-from"
+      - "type=gha,scope=amd64,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
+      - "--builder={{ .Env.BUILDX_BUILDER }}"
+    extra_files:
+      - "images/ansible-operator/Pipfile"
+      - "images/ansible-operator/Pipfile.lock"
+  - image_templates:
+      - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-arm64"
+    dockerfile: images/ansible-operator/Dockerfile
+    goos: linux
+    goarch: arm64
+    use: buildx
+    build_flag_templates:
+      - "--platform=linux/arm64"
+      - "--cache-from"
+      - "type=gha,scope=arm64,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
+      - "--builder={{ .Env.BUILDX_BUILDER }}"
+    extra_files:
+      - "images/ansible-operator/Pipfile"
+      - "images/ansible-operator/Pipfile.lock"
+  - image_templates:
+      - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-ppc64le"
+    dockerfile: images/ansible-operator/Dockerfile
+    goos: linux
+    goarch: ppc64le
+    use: buildx
+    build_flag_templates:
+      - "--platform=linux/ppc64le"
+      - "--cache-from"
+      - "type=gha,scope=ppc64le,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
+      - "--builder={{ .Env.BUILDX_BUILDER }}"
+    extra_files:
+      - "images/ansible-operator/Pipfile"
+      - "images/ansible-operator/Pipfile.lock"
+  - image_templates:
+      - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-s390x"
+    dockerfile: images/ansible-operator/Dockerfile
+    goos: linux
+    goarch: s390x
+    use: buildx
+    build_flag_templates:
+      - "--platform=linux/s390x"
+      - "--cache-from"
+      - "type=gha,scope=s390x,token={{ .Env.ACTIONS_RUNTIME_TOKEN }},url={{ .Env.ACTIONS_CACHE_URL }}"
+      - "--builder={{ .Env.BUILDX_BUILDER }}"
+    extra_files:
+      - "images/ansible-operator/Pipfile"
+      - "images/ansible-operator/Pipfile.lock"
```

Comment:
could we keep the indentation as before to make easier the review?

Reply:
Sure, I have reverted the indentation changes for easier review.